src/components/Text.js

```diff
@@ -84,10 +73,16 @@ const textStyles = {
     fontSize: 14,
     lineHeight: 20
   },
-  xSmall: {
-    fontFamily: "Roboto",
+  price: {
```

Comment:
With the classes in this component I was kinda trying to follow the names of typographic styles laid out by the designers here: https://app.zeplin.io/project/5a916701f75c524181eebe68/screen/5ab1041c5727d6b45590a18d

It's changed since I last wrote these though. I feel like it might be neater to stick with generic classes like `h1`, `para`, `small` etc and then let components like `Price` that use this component just pick which typographic style it needs. If the styles need to be different from what's available you can pick the closest one and add extra styles through the normal style prop like `<Text style={{includeFontPadding: false}}>`.

It's not really a big deal, just felt conceptually neater to me

Reply:
Yeah. We updated the styles here to match the ones in Zeplin again. Since the first designs are signed off now, I think hope also the text styles are mostly final.

Reply:
Also `price` and `tabBarItem` are now part of the Typography here: https://app.zeplin.io/project/5a916701f75c524181eebe68/screen/5ab1041c5727d6b45590a18d

Reply:
Oh I didn't see those. In that case ignore me then. If it matches the design guide I'd say that's fair.